app/models/authority.rb

```diff
@@ -261,7 +261,16 @@ def load_councillors(popolo)
 
     persons.map do |person|
       councillor = councillors.find_or_create_by(name: person.name)
-      councillor.update(email: person.email, image_url: person.image, party: person.party, popolo_id: person.id)
+
+      councillor.popolo_id = person.id
+
+      councillor.image_url = if councillor.cached_image_available?
```

Comment:
Using the ternary operator here will be more concise and clear. Of course if my suggestion about not falling back to the popolo url is accepted then this doesn't matter :smile: